content/en/serverless/google_cloud_run/_index.md

```diff
@@ -2,201 +2,234 @@
+Depending on your deployment process, push the image built to a registry. The `gcr.io` prefix in the examples above assumed [Google Cloud Registry][11].
 
-3. Run this command to submit your build to GCP.
+### Deploy the Application with the Datadog Agent
 
-   ```
-   gcloud builds submit --tag gcr.io/YOUR_PROJECT/YOUR_APP_NAME
-   ```
-4. Create a secret from your Datadog API key.
-   Go to [Secret Manager][5] in your GCP console and click on **Create secret**.
+{{< tabs >}}
 
-   Set a name (for example, `datadog-api-key`) in the **Name** field. Then, paste your Datadog API key in the **Secret value** field.
-5. Deploy your service.
-   Go to [Cloud Run][6] in your GCP console. and click on **Create service**.
+{{% tab "Cloud Run web UI" %}}
 
-   Select **Deploy one revision from an existing container image**. Choose your previously built image.
+1. Create a secret with Datadog API key. This can be done via [Secret Manager](https://console.cloud.google.com/security/secret-manager) in your GCP console and clicking on **Create secret**. Set a name (for example, `datadog-api-key`) in the **Name** field. Then, paste your Datadog API key in the **Secret value** field.
 
-   Select your authentication method.
+2. Go to [Cloud Run](https://console.cloud.google.com/run) in your GCP console. and click on **Create service**.
 
-   Reference your previously created secret. Go to the **Container, Variables & Secrets, Connections, Security** section and select the **Variables & Secrets** tab.
+3. Select **Deploy one revision from an existing container image**. Choose your previously built image.
 
-   Under **Secrets**, click on **Reference a secret** and choose the secret you created from your Datadog API key. You may need to grant your user access to the secret.
+4. Select your invocation authentication method.
 
-   Under **Reference method**, select **Exposed as environment variable**.
+5. Reference your previously created secret, named `datadog-api-key` in this guide. Go to the **Container, Networking, Security** section and select the **Secrets** tab. click on **Reference a secret** and choose the secret you created from your Datadog API key. You may need to grant your user access to the secret.
 
-   Under the **Environment variables** section, ensure that the name is set to `DD_API_KEY`.
+6. Under **Reference method**, select **Exposed as environment variable**.
 
-### Custom metrics
-You can submit custom metrics using a [DogStatsd client][7].
+7. Under the **Environment variables** section, ensure that the name is set to `DD_API_KEY`.
 
-**Note**: Only `DISTRIBUTION` metrics should be used.
+8. Still under **Environment variables**, create two more variables. One named `DD_TRACE_ENABLED` set to `true` to enable tracing. Another named `DD_SITE` containing the Datadog site which will collect the data (if not set, it defaults to `datadoghq.com`)
```

Comment:
You can actually do this: Set `<DD_SITE>` with {{< region-param key="dd_site" code="true" >}} (ensure the correct SITE is selected on the right).

The {{< region-param key="dd_site" code="true" >}} macro gives you the correct site value based what's selected in the doc site (note the site selector top right). Otherwise, it's hard for user to figure out the exact value for DD_SITE.